doc/man3/EVP_PKEY_CTX_ctrl.pod

```diff
@@ -274,17 +274,32 @@ buffer is expected to be the algorithm identifier byte.
 
 The EVP_PKEY_CTX_get_rsa_padding() function gets the RSA padding mode for B<ctx>.
 
-The EVP_PKEY_CTX_set_rsa_pss_saltlen() macro sets the RSA PSS salt length to
-B<len>. As its name implies it is only supported for PSS padding. Three special
-values are supported: B<RSA_PSS_SALTLEN_DIGEST> sets the salt length to the
-digest length, B<RSA_PSS_SALTLEN_MAX> sets the salt length to the maximum
-permissible value. When verifying B<RSA_PSS_SALTLEN_AUTO> causes the salt length
-to be automatically determined based on the B<PSS> block structure. If this
-macro is not called maximum salt length is used when signing and auto detection
-when verifying is used by default.
+The EVP_PKEY_CTX_set_rsa_pss_saltlen() function sets the RSA PSS salt
+length to B<saltlen>. As its name implies it is only supported for PSS
+padding. If this function is not then the called maximum salt length
```

Comment:
is not ??

Reply:
Today I learned "[gobbledygook](https://en.wiktionary.org/wiki/gobbledygook)". That's an interesting word and it sounds really funny, I'll have to remember it. ;-)

Reply:
You native speakers, you always use so many words which we foreign language speakers need to look up in the dictionary. This reminds me of a nice little online article in the [Financial Times](https://www.ft.com/content/bc8f9c5c-12a3-11ea-a225-db2f231cfeae?segmentId=09cf3415-e461-2c4a-a8cc-80acc4846679) which I happened to find this week via Twitter.  ;-)

Reply:
I love getting to learn new words (didn't happen this time, though), so I'm partial for getting confused :wink:

Reply:
Mee too, of course. Just kiddin'.

Reply:
Either way, I agree the sentence was awful.  Fixed.

Reply:
I'm flabbergasted that you managed to work so long on OpenSSL and didn't know of gobbledygook.  But I'm chuffed you know now :)

Reply:
Er, who're you talking to @richsalz?  If it's to me, you didn't read "(didn't happen this time, though)" properly :wink:

Reply:
@mspncp  Figure it couldn't hurt to add some new words :)